raiden/tests/integration/network/transport/utils.py

```diff
@@ -0,0 +1,14 @@
+import secrets
+
+from coincurve import PublicKey
+from eth_typing import Address
+from eth_utils import to_canonical_address, to_checksum_address
+from sha3 import keccak_256
+
+
+def generate_address() -> Address:
```

Comment:
Awesome, maybe we could return the private key too to use this outside comms also.